jina/peapods/runtimes/asyncio.py

```diff
@@ -125,22 +131,48 @@ def activate(**kwargs):
         # does not apply to this types of runtimes
         pass
 
+    @staticmethod
+    def is_ready(ctrl_address: str, **kwargs) -> bool:
+        """
+        Check if status is ready.
+
+        :param ctrl_address: the address where the control message needs to be sent
+        :param kwargs: extra keyword arguments
+
+        :return: True if status is ready else False.
+        """
+
+        try:
+            GrpcConnectionPool.send_message_sync(ControlMessage('STATUS'), ctrl_address)
```

Comment:
I am not sure that really belongs here? the grpc server is created by the child class implementations.

Reply:
But it is the only way ContainerPea can avoid knowing runtime_cls. I agree this should be done as discussed via the container itself, but not possible now